common/djangoapps/edxmako/paths.py

```diff
@@ -49,15 +54,17 @@ def add_directory(self, directory, prepend=False):
 
     def get_template(self, uri):
         """
-        Overridden method which will hand-off the template lookup to the microsite subsystem
+        Overridden method which will hand-off the template lookup to the theming subsystem
```

Comment:
Can we add a little bit more here about how this works, ie, the scenarios -- maybe cover the use cases where the lookup would not be necessary, as well as the valid case for the try, as well as the invalid case which would trigger the except.